docs/source/description.rst

```diff
@@ -2,3 +2,17 @@ Alot is a terminal-based mail user agent for the notmuch mail system.
 It features a modular and command prompt driven interface
 to provide a full MUA experience as an alternative to the Emacs mode shipped
 with notmuch.
+What you can do whit alot:
+
+	* Read mail/thread
+	* Search mail
+	* Tag/Untag mail
+	* Filter mail	
+	* Save attachment
+	* Receive mail (using external tool as `mbsync <http://isync.sourceforge.net/mbsync.html>`_ or `offlineIMAP <http://www.offlineimap.org/>`_)
```

Comment:
I am confused: Is there anything we support especially for receiving mail from withing alot? Because if it just the ability to run a shell script this is not a feature specific to receiving mail.